scss/styles.scss

```diff
@@ -6,5 +6,5 @@ $font-stack: 'Roboto', sans-serif;;
 }
 
 .container {
-  padding: 140px 100px;
+  padding: 140px 0 50px 100px;
```

Comment:
by doing this you have 0 to the right and 100px to the left, this prevents the elements to be correctly centered, use the same values for this here.